lib/screens/settings/settings.dart

```diff
@@ -93,6 +93,45 @@ class _SettingsState extends State<Settings> {
     );
   }
 
+  Widget _buildStammHeimInput() {
+    return ListTile(
+      title: TextField(
+        controller: _stammheimTextController,
+        decoration: const InputDecoration(
+          labelText: 'Stammheim Adresse',
+        ),
+      ),
+      trailing: Row(
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          IconButton(
+            color: Theme.of(context).colorScheme.primary,
+            onPressed: () async {
+              final text = _stammheimTextController.text;
+              setStammheim(text);
+              final scaffold = ScaffoldMessenger.of(context);
+              try {
+                final locations = await locationFromAddress(text);
```

Comment:
Ich frage mich in welchem Fall locationFromAddress mehrere Ergebnisse zurück liefert. Selbst wenn man nur Hauptstraße oder absichtlich fehlerhaftes eingibt kommt nur ein Ergebnis.

Reply:
Ja das stimmt. Es steht halt so in der Dokumentation der Methode, dass das selten passieren kann.